test/test_data/firefox.py

```diff
@@ -17,8 +17,8 @@
     },
     {
         "url": "http://ftp.fr.debian.org/debian/pool/main/f/firefox/",
-        "package_name": "firefox_106.0.5-1_arm64.deb",
+        "package_name": "firefox_83.0-1_mipsel.deb",
         "product": "firefox",
-        "version": "106.0.5",
+        "version": "83.0",
```

Comment:
Seems like the pattern is available on older versions of Firefox (which is fine btw, something is better than nothing)